piper/ratbagd.py

```diff
@@ -242,6 +247,13 @@ def buttons(self):
         return self._buttons
 
     @GObject.Property
+    def leds(self):
+        """
+        A list of RatbagdLed objects with this profile's leds.
+        """
```

Comment:
doesn't this need to be on one line to make flake8 happy? or am I misremembering?